nx.json

```diff
@@ -2,18 +2,15 @@
   "cli": {
     "packageManager": "bun"
   },
-  "cacheableOperations": ["build", "test", "check", "check:fix"],
```

Comment:
`cacheableOperations` is a legacy configuration option, `cache: true` as defined below is now the [recommended configuration](https://lerna.js.org/docs/features/cache-tasks#setup)